src/bus/mod.rs

```diff
@@ -1,7 +1,8 @@
-use anyhow::{bail, Ok, Result};
 #[cfg(unix)]
 use std::{env, path::Path};
 use std::{str::FromStr, sync::Arc};
+
+use anyhow::{bail, Ok, Result};
```

Comment:
also this change is unrelated to the commit. :) I wouldn't have even mentioned it but since you're going to update the commit anyway. :)